src/frontend/context/Eøs/EøsContext.tsx

```diff
@@ -53,11 +53,14 @@ export const sorterEøsPerioder = (
     if (periodeA.tom === undefined && !beggePerioderLøpende) return -1;
     if (periodeB.tom === undefined && !beggePerioderLøpende) return 1;
```

Comment:
```suggestion
    if (periodeA.tom === undefined && periodeB.tom !== undefined) return -1;
    if (periodeB.tom === undefined && periodeA.tom !== undefined) return 1;
```

Reply:
Da kan man også fjerne `beggePerioderLøpende`

Reply:
Eventuelt er det dette som er i prod nå: 
```suggestion
    if (periodeA.tom === undefined) return -1;
    if (periodeB.tom === undefined) return 1;
```

Reply:
✅